New Permission Request Documentation (#5771)
sourcegraph/handbook: content/departments/security/tooling/entitle.md

# Entitle

## What is it?

> Entitle is an authentication and access management system. It is a cloud based application that automates access requests, approvals, and revocation of permissions based on custom policies. All of this can be done via slack with a simple command “/entitle”.

Reviewer: ```suggestion
Entitle is an authentication and access management system. It is a cloud based application that automates access requests, approvals, and revocation of permissions based on custom policies. All of this can be done via Slack with a simple command “/entitle”.
```

nit

## Why Entitle?

After reviewing multiple options available on the market for permission management, Entitle stood out as a clear innovator in the field. Entitle has deep integrations with various applications that we currently use. We have been working closely with them since our first Sourcegraph Merge to get integrations setup and polish the user interface.

There are many security concerns that come with Identity and Access Management. Entitle helps address many of them for us, including:

> - No need to for indefinite access to sensitive information

Reviewer: ```suggestion
- No need for indefinite access to sensitive information
```

> - Auto expiration of access based on time bound requests

Reviewer: ```suggestion
- Auto expiration of access based on timebound requests
```

- Audit log of all requests and actions taken

> - Granular access means no more over provisioned permissions

Reviewer: ```suggestion
- Granular access means no more overprovisioned permissions
```

- Granular access means only access to relevant data

Entitle also gives us advantages over the current request process. It allows for:

> - Self serve request process

Reviewer: ```suggestion
- Self-service request process
```
